packages/contracts/test/exchange/wrapper.ts

```diff
@@ -38,49 +44,57 @@ contract('Exchange', (accounts: string[]) => {
     let orderFactory: OrderFactory;
 
     before(async () => {
-        [tokenRegistry, exchange] = await Promise.all([TokenRegistry.deployed(), Exchange.deployed()]);
-        exWrapper = new ExchangeWrapper(exchange);
-        const [repAddress, dgdAddress, zrxAddress] = await Promise.all([
-            tokenRegistry.getTokenAddressBySymbol('REP'),
-            tokenRegistry.getTokenAddressBySymbol('DGD'),
-            tokenRegistry.getTokenAddressBySymbol('ZRX'),
+        const accounts = await web3Wrapper.getAvailableAddressesAsync();
+        tokenOwner = maker = accounts[0];
+        taker = accounts[1] || accounts[accounts.length - 1];
+        feeRecipient = accounts[2] || accounts[accounts.length - 1];
```

Comment:
I really dislike these. So one environment is TestRPC where we have 10 accounts. What is the second one? And should these tests work for both the case where `taker` === `feeRecipient` and when not? Sounds error prone to me. @abandeali1

Reply:
Yeah you're right, I think a lot of these tests would probably fail if `taker` === `feeRecipient`